taxi/views.py

```diff
@@ -14,7 +14,7 @@ def index(request):
     num_drivers = Driver.objects.count()
     num_cars = Car.objects.count()
     num_manufacturers = Manufacturer.objects.count()
-    num_visits = request.session.get("num_visits", 1)
+    num_visits = request.session.get("num_visits", 0)
     request.session["num_visits"] = num_visits + 1
```

Comment:
still wrong because you show the old number of visits. you show the previous number of views but when use opens the page it should display a new number